projects/packages/publicize/src/class-services.php

```diff
@@ -7,15 +7,15 @@
 
 namespace Automattic\Jetpack\Publicize;
 
-use Automattic\Jetpack\Connection\Client;
-use Automattic\Jetpack\Connection\Manager;
+use Automattic\Jetpack\Publicize\REST_API\Proxy_Requests;
+use WP_REST_Request;
 
 /**
  * Publicize Services class.
  */
 class Services {
 
-	const SERVICES_TRANSIENT = 'jetpack_social_services_list';
+	const SERVICES_TRANSIENT = 'jetpack_social_available_services_list';
```

Comment:
It's important to change this to avoid older format being returned when it expects a new one following the usage of proxy helper which `json_decode`s as associate array instead of an object.